selfdrive/controls/controlsd.py

```diff
@@ -62,8 +61,8 @@ def __init__(self, sm=None, pm=None, can_sock=None):
       ignore = ['driverCameraState', 'managerState'] if SIMULATION else None
       self.sm = messaging.SubMaster(['deviceState', 'pandaState', 'modelV2', 'liveCalibration',
                                      'driverMonitoringState', 'longitudinalPlan', 'lateralPlan', 'liveLocationKalman',
-                                     'managerState', 'liveParameters', 'radarState'] + self.camera_packets,
-                                     ignore_alive=ignore, ignore_avg_freq=['radarState', 'longitudinalPlan'])
+                                     'managerState', 'liveParameters'] + self.camera_packets,
+                                     ignore_alive=ignore, ignore_avg_freq=['longitudinalPlan'])
```

Comment:
```suggestion
                                     ignore_alive=ignore)
```
this can be removed since it's now driven by modeld

Reply:
ok